Resources/Prototypes/Tiles/floors.yml

```
@@ -2264,4 +2272,5 @@
     collection: BarestepWood
   itemDrop: FloorTileItemWoodLarge
   heatCapacity: 10000
-  tileRipResistance: 100
\ No newline at end of file
+  tileRipResistance: 100
+  
```

```suggestion
  tileRipResistance: 100
```
maybe this will fix the whitespace fails

---

It was like that originally and gave a missing new line at end of file fail
So I really don't know